frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/UnlockedResponses.tsx

```diff
@@ -1,21 +1,45 @@
-import { useMemo } from 'react'
-import { Box, Flex, Grid, Text } from '@chakra-ui/react'
+import { useCallback, useMemo } from 'react'
+import { Box, Flex, Grid, Skeleton, Stack, Text } from '@chakra-ui/react'
 import simplur from 'simplur'
 
+import Button from '~components/Button'
 import Pagination from '~components/Pagination'
 
 import { DownloadButton } from './DownloadButton'
 import { ResponsesTable } from './ResponsesTable'
+import { SubmissionSearchbar } from './SubmissionSearchbar'
 import { useUnlockedResponses } from './UnlockedResponsesProvider'
 
 export const UnlockedResponses = (): JSX.Element => {
-  const { currentPage, setCurrentPage, count, isLoading } =
-    useUnlockedResponses()
+  const {
+    currentPage,
+    setCurrentPage,
+    count,
+    filteredCount,
+    isLoading,
+    submissionId,
+    isAnyFetching,
+    setSubmissionId,
+  } = useUnlockedResponses()
+
+  const countToUse = useMemo(() => {
+    if (submissionId) {
+      return filteredCount
+    }
+    return count
+  }, [filteredCount, count, submissionId])
 
   const prettifiedResponsesCount = useMemo(() => {
-    if (!count) return
-    return simplur` ${[count]}response[|s] to date`
-  }, [count])
+    if (filteredCount !== undefined) {
+      return simplur` ${[filteredCount]}result[|s] found`
+    } else if (count !== undefined) {
+      return simplur` ${[count]}response[|s] to date`
```

Comment:
Just checking: is the spacing correct?
```suggestion
      return simplur`${[filteredCount]} result[|s] found`
    } else if (count !== undefined) {
      return simplur`${[count]} response[|s] to date`
```

Reply:
it is, there is a number before this string haha

Reply:
Oh duh! Sorry, I thought `simplur` was including the number in the output as well. Thanks for the clarification.